.ci/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM centos:7
+
+RUN yum install -y epel-release make gcc krb5-devel python-devel
+RUN yum install -y python-pip
+RUN pip install koji
+
+WORKDIR /code
+COPY requirements* ./
+RUN pip install -r requirements-test.txt
+
+ENV PYTHONPATH /code/doozerlib
```

Comment:
could we also add:
```
ENV PYTHONDONTWRITEBYTECODE=1
```
since in a local test we're mounting in the code dir, don't want to write all the `.pyc` files.

Reply:
+1 adding now ;D

Reply:
Resolved in commit 52330677